File: proxy-httpclient/src/main/java/br/com/leonardoferreira/poc/httpclient/Application.java
package br.com.leonardoferreira.poc.httpclient;

import br.com.leonardoferreira.poc.httpclient.annotation.EnableHttpClient;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@EnableHttpClient
@SpringBootApplication
public class Application {

    public static void main(String[] args) {
        SpringApplication.run(Application.class, args);
    }
}

File: kafka/spring-cloud-stream-transaction/src/main/kotlin/br/com/leonardoferreira/poc/Application.kt
package br.com.leonardoferreira.poc

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.transaction.annotation.EnableTransactionManagement

@SpringBootApplication
@EnableTransactionManagement
class Application

fun main(args: Array<String>) {
    runApplication<Application>(*args)
}
